LICENCE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 The National Archives, UK
+Copyright (c) 2024 The National Archives, UK
```

Comment:
This is the licence for this repo. You don't need to keep the licence year up to date, you just keep the year that it was created which was 2023 for this one so you can revert this change.